### PR TITLE
Send environment variables in payload

### PR DESCRIPTION
**Pull request by zachgoldstein**
Closes: https://github.com/airbrake/airbrake-python/issues/52

### CODE REVIEW

File: airbrake/notifier.py [hunk 1]
```diff
@@ -96,6 +97,8 @@ def __init__(self, project_id=None, api_key=None, host=None, timeout=None,
         self.blacklist_keys = config.get("blacklist_keys", [])
```
Comment (samstav) [inline airbrake/notifier.py]: Do the other sdks have default blacklist keys? What about using regex?

Comment (zachgoldstein) [inline airbrake/notifier.py]: We do have regex filters in the ruby notifier, and we should definitely add them. I've added an issue for that here: https://github.com/airbrake/airbrake-python/issues/69.

I think some other sdks have default blacklisted keys, and ya we should probably add some like `username`, and `password`. I'll go and check out what consensus is for other systems.

**Comment by zachgoldstein** [comment]
Looks like this sort of feature has been pulled out of phpbrake and the ruby notifier. I'm starting to think that we should only do this if the user explicitly requests it. So we'd have a whitelist of keys that would be pulled out of the env and sent. I feel like this would be safe by default since it doesn't send anything, but still gives users the flexibility to do this.

**Comment by zachgoldstein** [comment]
I'm going to ditch this PR. If it's been pulled from the php and ruby notifiers, we probably shouldn't bother with it here. I'm going to keep a branch kicking around in case things change.

**Pull request closed without merging**